eatsmart/locations/durham/forms.py

```diff
@@ -80,6 +80,7 @@ class InspectionForm(forms.ModelForm):
 
     class Meta:
         model = Inspection
+        fields = "__all__"
```

Comment:
Let's just figure out which fields we want to really allow the user to edit.
https://docs.djangoproject.com/en/1.8/topics/forms/modelforms/#selecting-the-fields-to-use